docs/src/design-system/dividers/DividerUsagePage.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import DividerUsagePage from './DividerUsagePage';
+
+export default {
+  title: 'Design System / Dividers',
```

Comment:
Since this is the only divider documentation it doesn't need its own folder.  Instead you can have ` title: 'Design System'`.  

Then you can add `DividerUsage.storyName = 'Dividers';` so it matches the same naming pattern of the other component docs.